src/sms-password/templates/user_management/sms-password/reset_form.html

```diff
@@ -34,8 +33,8 @@
         <div class="help-block" ng-show="errors.new_password2" ng-bind="errors.new_password2"></div>
     </div>
 
-    <div class="help-block" ng-if="errors.detail" ng-bind="errors.detail"></div>
-
     <button type="submit" class="btn btn-default" ng-click="changePassword()" translate>Update password</button>
 </div>
 
+<div class="has-error" ng-if="errors.detail" ng-bind="errors.detail"></div>
+<div class="has-error" ng-if="errors.non_field_errors" ng-bind="errors.non_field_errors"></div>
```

Comment:
You don't have to move them out of the `ng-show="requested"` block, they can be in that block. At that point the diff will show just indentation change for all elements being indented into that block